Disaster Recovery/SafeHaven 5 General/resize-linux-pg.md

```diff
@@ -0,0 +1,239 @@
+```
+In this example, the iSCSI disk path is `/dev/sdd`.
+
+Secondly, we could run the `lsblk` command to check its disk size. It is possible that the disk size is not changed until a SCSI bus rescan. In this case, we could run 
+
+```bash
+echo 1 > /sys/class/scsi_device/3\:0\:0\:0/device/rescan
+```
+
+In this example, the size of `sdd` changes from 26GB to 30GB after the above rescan and we have
+
+```bash
+root@WA1SHSJ-U18-01:/opt/datagardens/rootfs/mnt/fs-disk1# lsblk
+NAME                   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+sda                      8:0    0   10G  0 disk 
+`-sda1                   8:1    0  487M  0 part /boot
+sdb                      8:16   0    2G  0 disk [SWAP]
+sdc                      8:32   0   14G  0 disk /
+sdd                      8:48   0   30G  0 disk 
+|-sdd1                   8:49   0    1M  0 part 
+|-sdd2                   8:50   0  487M  0 part 
+`-sdd3                   8:51   0 25.5G  0 part 
+  |-vg_iscsi18-swap_lv 253:0    0    2G  0 lvm  
+  `-vg_iscsi18-root_lv 253:1    0 23.5G  0 lvm  /opt/datagardens/rootfs
+sr0                     11:0    1 1024M  0 rom  
+```
+
+#### 2.2 Expand the root File System Partition
+
+There are two different cases depending on the DR cloud type.
+
+##### 2.2.1 For DR to VMware/CLC/vCloud: Expand Logical Volume
+
+In this case, the iSCSI is partition to support logical volume manager (LVM). We could check its disk partition with the `gdisk` tool as
+
+```bash
+root@WA1SHSJ-U18-01:/opt/datagardens/rootfs/mnt/fs-disk1# gdisk -l /dev/sdd
+GPT fdisk (gdisk) version 1.0.3
+
+Partition table scan:
+  MBR: protective
+  BSD: not present
+  APM: not present
+  GPT: present
+
+Found valid GPT with protective MBR; using GPT.
+Disk /dev/sdd: 62914560 sectors, 30.0 GiB
+Model: disk_00         
+Sector size (logical/physical): 512/512 bytes
+Disk identifier (GUID): E4D3AF94-DE60-4E74-9420-56CA8DA39F74
+Partition table holds up to 128 entries
+Main partition table begins at sector 2 and ends at sector 33
+First usable sector is 34, last usable sector is 54525918
+Partitions will be aligned on 2048-sector boundaries
+Total free space is 4061 sectors (2.0 MiB)
+
+Number  Start (sector)    End (sector)  Size       Code  Name
+   1            2048            4096   1024.5 KiB  EF02  grub bios boot
+   2            6144         1003519   487.0 MiB   8300  boot partition
+   3         1003520        54525918   25.5 GiB    8E00  PV for vg_iscsi18
+```
+
+And the `pvs` command will confirm that the `/dev/sdd3` parition is the physical volume used for the iSCSI target.
+
+```bash
+root@WA1SHSJ-U18-01:/opt/datagardens/rootfs/mnt/fs-disk1# pvs
+  PV         VG         Fmt  Attr PSize   PFree
+  /dev/sdd3  vg_iscsi18 lvm2 a--  <25.52g    0 
+```
+
+We could use the `parted` command to resize the partition 3.
+
+First time launching `parted /dev/sdd` will have a warning message since the disk size has changed and the GPT partition table needs fixing. Please enter `Fix` to fix it.
```

Comment:
Perhaps a comma, e.g.
> will display a warning message, since the disk size changed - the GPT partition table therefore needs to be updated. Please enter `Fix` to update it.